okio-files/src/commonMain/kotlin/okio/Filesystem.kt

```diff
@@ -28,6 +28,16 @@ abstract class Filesystem {
   @Throws(IOException::class)
   abstract fun cwd(): Path
 
+  /**
+   * Returns the children of the directory identified by [path], or null if [path] is a not a
```

Comment:
nit: either rename the parameter to `path` or change the doc to `dir`

Reply:
good catch. Fixed.